core/src/main/java/no/nordicsemi/android/kotlin/ble/core/BleDevice.kt

```diff
@@ -75,16 +75,16 @@ sealed interface BleDevice : Parcelable {
      * Returns true if a device has not empty name.
      */
     val hasName
-        get() = name.isNotEmpty()
+        get() = name?.isNotEmpty() == true
 }
 
 /**
  * Class representing BLE server device. It can be either mocked or native variant.
- * It can be connected to using [ClientBleGatt].
+ * It can be connected to using [ClientDevice].
```

Comment:
ClientBleGatt is used to create connections

Reply:
Sorry, it's jus not visible from that module and AS shows as warning, at least for me.